lib/config/options/index.ts

```diff
@@ -2844,6 +2844,13 @@ const options: RenovateOptions[] = [
     cli: false,
     env: false,
   },
+  {
+    name: 'milestone',
+    description: `The number of a milestone. If set, the milestone will be set on the PR.`,
+    type: 'integer',
```

Comment:
Would having a array of milestones make sense? Where a Renovate PR would help multiple milestones? I'm thinking of a case like this:

- Milestone 10, update LTS version of program
- Milestone 11, update next version of program

Reply:
I'm not sure if I can follow here. 
At least in GitHub (I am fairly certain) every issue/pr can only be part of one single milestone. Or am I missing something?

Reply:
Let's keep it single